### PR TITLE
PRT-987 SDK - Implement batch requests

### PR DESCRIPTION
**Pull request by shleikes**

### CODE REVIEW

File: ecosystem/lava-sdk/src/chainlib/common.ts [hunk 1]
```diff
@@ -23,3 +24,46 @@ export function getChainParser(apiInterface: string) {
...
+// NOT_APPLICABLE
+// LATEST_BLOCK
+// PENDING_BLOCK
+// SAFE
+// FINALIZED
+// numeric value (descending)
+// EARLIEST
+//
+// returns: [latest, earliest]
+export function compareRequestedBlockInBatch(
```
Comment (omerlavanet) [inline ecosystem/lava-sdk/src/chainlib/common.ts:39]: let's verify this with a unitest

File: ecosystem/lava-sdk/src/sdk/sdk.ts [hunk 2]
```diff
@@ -506,27 +509,31 @@ export class LavaSDK {
...
+        }
+        const dec = new TextDecoder();
+        const decodedResponse = dec.decode(reply.getData_asU8());
+        // Parse response
+        const jsonResponse = JSON.parse(decodedResponse);
+        // Return response
+        return jsonResponse;
+      })
+      .catch((err) => {
+        console.log(err);
```
Comment (omerlavanet) [inline ecosystem/lava-sdk/src/sdk/sdk.ts:527]: we need to throw or return just printing is not enough

**Review by omerlavanet — CHANGES_REQUESTED** [review]

File: ecosystem/lava-sdk/src/sdk/sdk.ts (hunk 2 above)
Comment (shleikes) [reply ecosystem/lava-sdk/src/sdk/sdk.ts:527]: I think it was a log for myself. Removing

File: ecosystem/lava-sdk/src/chainlib/common.ts (hunk 1 above)
Comment (shleikes) [reply ecosystem/lava-sdk/src/chainlib/common.ts:39]: Done

**Review by omerlavanet — APPROVED** [review]

**Pull request merged**